app.js

```diff
@@ -31,9 +31,36 @@
 // 74 - Soft Tek Front
```

Comment:
We can remove the Doumbek comments since we moved the info to `soundfonts.json`.